docs/source/reference/function-intercepts.md

```
@@ -0,0 +1,573 @@
+5. Object stores
+6. Retrievers
+7. TTC strategies
+8. **Function intercepts** ← Built here
+9. Function groups
+10. Functions ← Use intercepts here
+
+## Troubleshooting
+
+### Common Issues
+
+**Intercept not found error**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)